primitives/core/src/assertion.rs

```diff
@@ -210,19 +210,15 @@ pub enum Assertion {
 	#[codec(index = 13)]
 	Oneblock(OneBlockCourseType),
 
-	// Sora Quiz
-	#[codec(index = 14)]
-	SoraQuiz(SoraQuizType),  // (sora_quiz_type)
-
 	// GenericDiscordRole
-	#[codec(index = 15)]
+	#[codec(index = 14)]
```

Comment:
have already released out the client api? I think the purpose of adding index is to have it with a fixed value. We don't change it very often.

Reply:
This PR deleted SoraQuiz and merged it into GenericDiscordRole so we have to re number the index

Reply:
Hmm, I don't think so we have to re number the index, it's up to us.

This is a breaking change - that's why it's always better to do all api changes within one pull request. It will probably won't be noticed if released together.

I suggest keeping old indices for `BnbDomainHolding` and `BnbDigitDomainClub`. By doing so, any software which relies on them can keep on operating - not very likely currently but that's a good practice overall.

You probably forgot to update `litentry-parachain/tee-worker/client-api/parachain-api/prepare-build/interfaces/vc/definitions.ts`, don't you ?